ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -13,8 +13,6 @@ if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
   exit 1
 fi
 
-echo "Installing from release ${RELEASE_IMAGE_LATEST}"
```

Comment:
This seems orthogonal to the thrust of this PR and [the `echo`][1] dates back to `ipi-conf-commands.sh` from #6708.  Can we spin this hunk out into a separate PR and get it landed while this one gets iterated on?

[1]: https://github.com/openshift/release/pull/6708/files#diff-50b0e5c1da4c6746d68864286c7c2876R40

Reply:
Ah, reading down I see you adding a similar `echo` in `ipi-install-install-commands.sh`, so I'm meh on whether we pull this out into a separate PR or not.  And while I'm talking about `ipi-install-install-commands.sh`, seems like a chance to remind folks that the stutter-removing PR is alive again: #7931.